tests/services/test_storage.py

```diff
@@ -0,0 +1,58 @@
+import shutil
+from pathlib import Path
+
+import pytest
+from tad.services.storage import WriterFactory
+from yaml import safe_load
+
+
+@pytest.fixture()
+def setup_and_teardown():
```

Comment:
You could use the default tmp_path fixture embeded in pytest. see list of fixtures here: https://docs.pytest.org/en/6.2.x/fixture.html

Reply:
oeh i like this one :) but i will not implement it, as the file system writer itself expects a location instead of a path which makes it more annoying to use tmp_path